source-code/website/src/pages/index/Header.tsx

```diff
@@ -1,189 +1,108 @@
 import { createSignal, For, Show } from "solid-js";
+import IconClose from "~icons/material-symbols/close-rounded";
+import IconMenu from "~icons/material-symbols/menu-rounded";
+import IconTwitter from "~icons/cib/twitter";
+import IconGithub from "~icons/cib/github";
 
 export function Header() {
 	const links = [
 		{ name: "Docs", href: "/documentation" },
 		{ name: "Editor", href: "/editor" },
 	];
 
-	const [show, setShow] = createSignal(false);
```

Comment:
renamed `show, setShow` to `mobileMenuIsOpen, setMobileMenuIsOpen`

It took several minutes to grasp what is supposed to be shown if `show` is `true`. The markup referenced `show` several times which contradicted my assumption that the mobileMenu should be shown.